engine/src/main/java/org/terasology/config/NetworkConfig.java

```diff
@@ -75,14 +73,16 @@ public String getServerMOTD() {
         try {
             StringBuffer mOTD = new StringBuffer();
 
-            FileReader reader = new FileReader("motd.md");
+            InputStreamReader reader = new InputStreamReader(new FileInputStream("motd.md"), "UTF8");
             BufferedReader bufferedReader = new BufferedReader(reader);
 
             String line;
             while ((line = bufferedReader.readLine()) != null) {
                 mOTD.append(line);
             }
 
+            reader.close();
+
```

Comment:
If an exception occurs between reader creation and this line then it won't be closed.

There is the autoclosable functionallity you can go for. E.g.
```try(InputStream inputStream = Files.newInputStream) {}```
or old style:
```
InputStream inputStream = ...;
try {

} finally {
   inputStream.close();
}
```
However since you are just reading all lines you can use `Files.readAllLines`.

I also noticed that the motd file has the extension `md`. Do we have highltighting for that already? If not do we have support for simple html? Maybe we should go for html then. Or even better put it in a module and then we are not limited.

Reply:
- I moved the `reader.close()` in a `finally` block.
- If the file is relatively larger in size won't the `Files.readAllLines()` cause problems? We can't be sure about the size of the message to be displayed afaik.
- Had a discussion with @Cervator regarding the` .html` or `.md` issue but currently there is support for neither of them so it did not matter when I implemented this.
- The MOTD should be displayed to all users when they join the game, so if this is moved to a module a user with that module deactivated would have no idea about the MOTD unless there is a feature present to activate this module by default for every user.